src/core_plugins/elasticsearch/index.js

```diff
@@ -107,7 +107,20 @@ export default function (kibana) {
         sslVerify(),
         rename('tribe.ssl.ca', 'tribe.ssl.certificateAuthorities'),
         rename('tribe.ssl.cert', 'tribe.ssl.certificate'),
-        sslVerify('tribe')
+        sslVerify('tribe'),
+        (settings = {}, log) => {
+          const tribe = get(settings, 'tribe');
+          if (!tribe) {
+            return;
+          }
+
+          const tribeKeys = Object.keys(tribe);
```

Comment:
iirc monitoring will have tribe settings too, this may need to be duplicated there

Reply:
I can't any settings in management for tribe. I do see that the security/management UI uses the `esDataIsTribe` angular global, but that gets injected by the elasticsearch plugin based on the settings deprecated here.

Is there something I'm missing?

Reply:
whoops, i thought i left a comment and resolved.  no i'm just throwing you for a loop, sorry about that.  you got it.